src/views/view-company-information.njk

```diff
@@ -0,0 +1,115 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+  {{
+    govukBackLink({
+      text: "Back",
+      href: Paths.SEARCH_COMPANY_URI
+    })
+  }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% include 'components/errors/error-summary.njk' %}
+
+      <h1 class="govuk-heading-xl">View company information</h1>
+      {% if not company.canClose %}
+      <div id="cannot-close-error" class="govuk-inset-text">
+        <h2 class="govuk-heading-l">You cannot use this service</h2>
+        <p class="govuk-body">This could be because the company:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>is not active</li>
```

Comment:
'- has no active directors or members' bullet point missing

Reply:
https://companieshouse.atlassian.net/browse/S4-56
see Jayti's comment, this is intentional